v3io/dataplane/kv.py

```diff
@@ -252,6 +255,8 @@ def scan(
             The container on which to operate.
         table_path (Required) : str
             The full path of the table
+        table_name (Optional) : str
+            The name of the table. Allows for specifying all tables with '/*'. If used, table_path must be an empty
```

Comment:
```suggestion
```suggestion
            The name of the table. Allows for specifying all tables with '/*'. If used, table_path must be an empty
            string.
```
```